tests/utils/crc32.test.ts

```diff
@@ -1,11 +1,14 @@
-import {calculateCrc32, Crc32Type} from '../../src/utils/calculateCrc32.js';
+import {calculateCrc32} from '../../src/utils/calculateCrc32.js';
+import getBytesFromHex from '../../src/utils/getBytesFromHex.js';
 
 
 describe('check crc32 calculation', () => {
     test('check crc32 IEEE_8023 calculation', () => {
```

Comment:
```suggestion
    test('IEEE_8023', () => {
```
?

and in other tests

Reply:
fixed